packages/flutter/lib/src/widgets/widget_span.dart

```diff
@@ -85,6 +87,37 @@ class WidgetSpan extends PlaceholderSpan {
         ),
       );
 
+  /// Helper function for extracting [WidgetSpan]s in preorder, from the given
+  /// [InlineSpan] as a list of widgets.
+  ///
+  /// The extracted widgets will be annotated with necessary semantics
```

Comment:
What's "necessary"? Necessary for what? :D

Reply:
Removed.